b2share/modules/communities/models.py

```diff
@@ -134,7 +137,11 @@ def receive_before_insert(mapper, connection, target):
             community=str(target.id),
             old_state=PublicationStates.submitted.name,
             new_state=PublicationStates.draft.name
-        )
+        ),
+        # permission to update the metadata of a published record
+        update_record_metadata_need_factory(
+            community=str(target.id),
+        ),
```

Comment:
@nharraud Can you tell me what's the best way to add this `record_metadata_need` to the community administrators already in the databases at CSC?